tests/behat/behat.yml

```diff
@@ -25,6 +27,7 @@ default:
       goutte: ~
       default_session: 'goutte'
       show_cmd: xdg-open %s
+      files_path: .
```

Comment:
For what is this necessary?
If generally necessary for working with files, please add it to our behat.yml in drunomics/drupal-project